src/ember-app/ember-cli-build.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env node */
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function(defaults) {
+  let app = new EmberApp(defaults, {
+		autoRun: false,
```

Comment:
We use `autoRun: false` because we don't want Ember to initialize immediately, but rather during the [single-spa `mount` lifecycle](https://github.com/CanopyTax/single-spa/blob/master/docs/child-applications.md#mount)